tsdef/LocalTrackPublication.d.ts

```diff
@@ -0,0 +1,15 @@
+import { Track } from './Track';
+import { TrackPublication } from './TrackPublication';
+
+export class LocalTrackPublication extends TrackPublication {
+  isTrackEnabled: boolean;
+  kind: Track.Kind;
+  priority: Track.Priority;
+  track: LocalTrack;
```

Comment:
Please export
```ts
export type LocalTrack = LocalAudioTrack | LocalDataTrack | LocalVideoTrack;
```
from a new file called `types.d.ts` and import it here. Otherwise `LocalTrack` will refer to an undefined type.